templates/CRM/common/l10n.js.tpl

```diff
@@ -13,7 +13,8 @@
   // Config settings
   CRM.config.userFramework = {$config->userFramework|@json_encode};
   CRM.config.resourceBase = {$config->userFrameworkResourceURL|@json_encode};
-  CRM.config.packagesBase = {capture assign=packagesBase}{crmResURL expr='[civicrm.packages]/.'}{/capture}{$packagesBase|@json_encode};
+  CRM.config.packagesBase = {capture assign=packagesBase}{crmResURL expr='[civicrm.packages]'}{/capture}{$packagesBase|@json_encode};
+  CRM.config.packagesBase = CRM.config.packagesBase.replace(/\/+$/, '') + '/';
```

Comment:
Stylistically, I like this patch better than #16779, but yeah - as you found, the `[civicrm.packages]` doesn't evaluate as expected. 

Since the output before was expected to end in a trailing slash, I can't help thinking that this might have been a one-character patch - just changing `/.` to `/`.

Reply:
Yeah it's unfortunate I had some local issue that made it seem like it worked at first. I think for the regression though at this point it was more about not making everyone test/review things again on all cms's.